registry/server/appRoutes/interfaces/index.ts

```diff
@@ -69,20 +97,20 @@ const conditionSpecialRole = {
     otherwise: Joi.required(),
 };
 
-export const appRouteSchema = Joi.object({
+export const appRouteSchema = Joi.object<AppRouteDto>({
     ...commonAppRoute,
     orderPos: commonAppRoute.orderPos.when('specialRole', {
         is: Joi.exist(),
         then: Joi.forbidden(),
-        otherwise: Joi.optional().default(null),
+        otherwise: Joi.optional(),
     }),
     route: commonAppRoute.route.when('specialRole', conditionSpecialRole),
     next: commonAppRoute.next.when('specialRole', {
         is: Joi.exist(),
         then: Joi.forbidden(),
     }),
 }).external(async (value) => {
-    if (value.orderPos === null) {
+    if (value.orderPos === undefined && !isPostgres(db)) {
```

Comment:
can we rename flag `isPostgres` in this context? I think that naming should target specific reason why for postgres this conditional is not applicable